src/styles/main.scss

```diff
@@ -0,0 +1,17 @@
+@import "./cardBlocks/card.scss";
+@import "./cardBlocks/stars.scss";
+@import "./variables.scss";
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  margin: 0;
+}
+
+p {
+  margin: 0;
+}
```

Comment:
Consider moving reset styling to a separate file to keep your main file clean and consistent